webpack-server.js

```diff
@@ -0,0 +1,21 @@
+var webpack = require('webpack')
+var WebpackDevServer = require('webpack-dev-server')
+var config = require('./webpack.config')
+
+new WebpackDevServer(webpack(config), {
+  publicPath: config.output.publicPath,
+  hot: true,
+  inline: true,
+  stats: { colors: true },
+  historyApiFallback: true,
+  watchOptions: {
+    aggregateTimeout: 300,
+    poll: 1000
+  },
+}).listen(3000, '0.0.0.0', function (err, result) {
```

Comment:
Folks developing on Linux won't like the `0.0.0.0` (but there's little to be done about it).